src/core/load_balancing/rls/rls.cc

```diff
@@ -754,7 +757,10 @@ class RlsLb final : public LoadBalancingPolicy {
   void UpdatePickerLocked() ABSL_LOCKS_EXCLUDED(&mu_);
 
   void MaybeExportPickCount(
-      GlobalInstrumentsRegistry::GlobalUInt64CounterHandle handle,
+      GlobalInstrumentsRegistry::TypedGlobalInstrumentHandle<
```

Comment:
Is there a way we can avoid writing this type out explicitly?

Reply:
I'm not sure and I don't think this is a big concern though. This type is part of the templated API and the caller should be able to referring to it directly.

Reply:
Done. Chose the template approach.